planemo/galaxy/activity.py

```diff
@@ -246,25 +277,18 @@ def stage_in(ctx, runnable, config, job_path, **kwds):  # noqa C901
     )
 
     if datasets and kwds.get("check_uploads_ok", True):
-        ctx.vlog("uploaded datasets [%s] for activity, checking history state" % datasets)
+        ctx.vlog(f"Uploaded datasets [{datasets}] for activity, checking history state")
         final_state = _wait_for_history(ctx, user_gi, history_id)
-
-        for (dataset, path) in datasets:
```

Comment:
Maybe this should stay, but do `if ctx.verbose` ?

Reply:
In https://github.com/galaxyproject/galaxy/pull/14963/files#diff-bf58520ed2b280d7e27f164c623c82481eb0de4a37d195dc59f57b10c9861a8fR156 I've changed the second member of the tuple returned by `galactic_job_json()` / `StagingInterface.stage()` to be a list of dataset dictionaries instead of a (dataset dict, UploadTarget), so this looked more effort than it's worth to support.